bot/gif_command.go

```diff
@@ -45,13 +45,13 @@ func NewGIFCommand(endpoint, key string, w io.Writer) cli.Command {
 			query := url.Values{}
 			query.Set("key", key)
 			query.Set("q", strings.Join(args, " "))
-			query.Set("limit", "1")
+			query.Set("mediafilter", "minimal")
 			if !c.Bool("explicit") {
 				query.Set("safesearch", "strict")
```

Comment:
Looking at their [docs](https://tenor.com/gifapi/documentation#endpoints-search): this should be a string flag, and in the usage we show the options: 'off', 'moderate', 'mild', or 'strict'

Reply:
Yeah but in practice I think people use it more in a binary sense. I don't think anyone will use it with those flags IMO, I just want it strict or off.

Reply:
Good point - let's keep the bool flag then. 

Now that I'm looking at it, I think this needs to be:
```go
query.Set("safesearch", "strict")
if c.Bool("explicit") {
    query.Set("safesearch", "off")
}
```